backend/src/database/provider_platforms.go

```diff
@@ -48,6 +49,18 @@ func (db *DB) CreateProviderPlatform(platform *models.ProviderPlatform) (*models
 	if err := db.Conn.Create(&platform).Error; err != nil {
 		return nil, err
 	}
+	if platform.Type == models.Kolibri {
```

Comment:
If a kolibri provider is added, it will create an OpenContentProvider automatically, which will eventually display it in the UI sidebar, presumably among others